Makefile

```diff
@@ -167,6 +167,7 @@ clean:
 	find . -name '*.created' -exec rm -f {} +
 	find . -name '*.pyc' -exec rm -f {} +
 	rm -rf certs *.tar vendor $(NODE_CONTAINER_BIN_DIR)
+	rm -rf filesystem/included-source/*
```

Comment:
Is that better in some way than `rm -rf filesystem/included-source` ?  I find the `*` a little uncomfortable there, although I don't think it introduces any real problem.